asserts/gpgkeypairmgr.go

```diff
@@ -23,15 +23,51 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"os"
 	"os/exec"
+	"path/filepath"
+	"strconv"
 	"strings"
+
+	"github.com/snapcore/snapd/osutil"
 )
 
+func ensureGPGHomeDirectory(homedir string) (string, error) {
```

Comment:
wondering if we still need to support passing in a homedir here? given the strong preference for the default location (and we have the env var for tests if needed)